src/pages/dashboard/charts/_Map.js

```diff
@@ -269,33 +274,58 @@ const Tooltip = ({ hoveredState, currentDate, getValue }) => {
   return (
     <div id="map-tooltip" style={{ top: y, left: x }}>
       <table>
+        <caption>
+          {d.properties.NAME}
+          <br />
+          <span className="date">{formatDate(parseDate(currentDate))}</span>
+        </caption>
         <thead>
           <tr>
-            <td colSpan="3">
-              {d.properties.NAME}
-              <br />
-              <span className="date">{formatDate(parseDate(currentDate))}</span>
-            </td>
+            <th scope="col">Metric</th>
```

Comment:
Also wanted to note why I added "Metric" to this previously empty cell. Per https://webaim.org/techniques/tables/data, 

> Table headers should never be empty. This is particularly of concern for the top-left cell of some tables

Also and more practically, when I was switching this from `<td />` to `<th />` there was a linting rule complaining at me :)